Xamarin.Essentials/DeviceInfo/DeviceInfo.ios.tvos.watchos.cs

```diff
@@ -62,6 +65,16 @@ static DeviceIdiom GetIdiom()
 #endif
         }
 
+        static Task<List<StorageInfo>> PlatformGetStorageInformation()
+        {
+            var attributes = NSFileManager.DefaultManager.GetFileSystemAttributes(Environment.GetFolderPath(Environment.SpecialFolder.Personal));
+            return Task.FromResult(
+                new List<StorageInfo>
+                {
+                    new StorageInfo(attributes.Size, attributes.FreeSize, attributes.Size - attributes.FreeSize, StorageType.Internal)
```

Comment:
Does iOS not have any external storage options?

Reply:
As far as I am aware it does not offer native external storage solutions.